### ISSUE TITLE
Test error: Cannot find the Button's aria-disabled attribute

### ISSUE THREAD

**Issue by dlabrecq** [patternfly/patternfly-react#10928]
**Please describe the issue**

2 tests cannot find the Button's `aria-disabled` attribute. This test worked with v5, but no longer works with v6.

**Are there visuals for this issue? Please provide screenshots**

Error 1
![Screenshot 2024-09-16 at 5 06 37 PM](https://github.com/user-attachments/assets/08568e86-8ab0-4408-b539-fc66a818a384)

Error 2
![Screenshot 2024-09-16 at 5 06 54 PM](https://github.com/user-attachments/assets/9c0d2877-b792-449d-a379-f822f44ef481)

**Could you point to a branch or draft PR where this issue exists? Please provide a link to the code**

Error 1
- Test https://github.com/dlabrecq/koku-ui/blob/pf-v6/src/routes/settings/costModels/components/addPriceList.test.tsx#L186
- Button https://github.com/dlabrecq/koku-ui/blob/pf-v6/src/routes/settings/costModels/components/addPriceList.tsx#L63

Error 2
- Test https://github.com/dlabrecq/koku-ui/blob/pf-v6/src/routes/settings/costModels/components/addPriceList.test.tsx#L244
- Button https://github.com/dlabrecq/koku-ui/blob/pf-v6/src/routes/settings/costModels/components/addPriceList.tsx#L63

**Comment by wise-king-sullyman** [comment]
These failures are in part because of the same cause as #10927, and in part because [we have updated the `isDisabled` prop for Button to set only `disabled` for button elements and not `aria-disabled`](https://github.com/patternfly/patternfly-react/pull/10255).

Along with my suggestion in that other issue I would suggest using the `toBeDisabled` matcher from jest-dom rather than checking for `aria-disabled` explicitly for these tests.

**Comment by dlabrecq** [comment]
It's unclear why the previous developer chose to test `aria-disabled`, but changed to use `toBeDisabled`
